tests/unit/aiplatform/test_lro.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020, Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gopgle.api_core.tests.unit import test_operation
+from google.cloud.aiplatform import base
+from google.cloud.aiplatform import lro
+from google.protobuf import empty_pb2 as empty
+
+TEST_OPERATION_NAME = "test/operation"
+
+
+def make_lro():
+    operation = test_operation.make_operation_future()
```

Comment:
probably not a good idea to import a helper function from api_core's unit tests.

Reply:
how so?

Reply:
figured out why